using-plugin-lib/.drone.yml

```diff
@@ -0,0 +1,376 @@
+---
+kind: pipeline
+type: vm
+name: testing
+
+pool:
+  use: ubuntu
+
+steps:
+- name: lint
+  image: {{ .input.build_image }}
+  pull: always
+  commands:
+  - go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
+  - golangci-lint version
+  - golangci-lint run
+  volumes:
+  - name: gopath
+    path: "/go"
+- name: test
+  image: {{ .input.build_image }}
+  commands:
+  - go test -cover ./...
```

Comment:
we should add 
```
go vet ./...
```
while we are here

Reply:
I was under the impression that golang CI runs that.

https://golangci-lint.run/usage/linters/#govet